### DISCUSSION TITLE
donate

### DISCUSSION THREAD

**Discussion by giscus** [luweizheng/lulaoshi.info#77] [Announcements]
# donate

如果你觉得本站内容还不错，可以打赏赞助一下作者。 微信支付宝 -- |

https://lulaoshi.info/donate.html

**Comment by zxy126** [comment]
从动手学深度学习而来，鲁老师文章解释了线性回归损失函数求导，对我有帮助。

**Comment by youhujun** [comment]
鲁大师官网博客写的很棒,受教了.已经赞了10元.我叫游鹄君,希望有机会可以结识鲁大师

**Reply by luweizheng** [reply]
我的邮箱 luweizheng36 AT hotmail.com 联系我邮箱，我把wx发给你
